### DISCUSSION TITLE
MVC file not generated due to Serenity.Pro.Coder issue

### DISCUSSION THREAD

**Discussion by theSTARTCenter** [serenity-is/Serenity#6864] [Help]
Hello,

We have tried many things to get the Serenity.Pro.Coder to generate the MVC file needed to alleviate these errors, but not having any luck.

> 
> Severity               Code     Description         Project File         Line        Suppression State
> 
> Error      CS0103  The name 'MVC' does not exist in the current context     xenoCore.Web                C:\Users\username\Desktop\xenoCore\xenoCore.Web\Modules\Common\GridPage\GridPageExtensions.cs         28           Active
> 
>  
> 
> Severity               Code     Description         Project File         Line        Suppression State
> 
> Warning               CS8034  Unable to load Analyzer assembly C:\Users\username\.nuget\packages\serenity.pro.coder\6.5.4\analyzers\dotnet\cs\Serenity.Pro.Coder.dll : Could not load file or assembly 'file:///C:\Users\username\.nuget\packages\serenity.pro.coder\6.5.4\analyzers\dotnet\cs\Serenity.Pro.Coder.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)                xenoCore.Web C:\Users\username\Desktop\xenoCore\xenoCore.Web\CSC              1              Active
> 
>  
> 
> Severity               Code     Description         Project File         Line        Suppression State
> 
> Warning               CS8034  Unable to load Analyzer assembly C:\Users\username\.nuget\packages\serenity.pro.coder\6.5.4\analyzers\dotnet\cs\Serenity.Pro.Coder.dll: Could not load file or assembly 'file:///C:\Users\username\AppData\Local\Temp\VS\AnalyzerAssemblyLoader\283f9f009ad54c9598c19ac50e5db539\2\Serenity.Pro.Coder.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)       xenoCore.Web                 1              Active
> 
>  
> 
> Severity               Code     Description         Project File         Line        Suppression State
> 
> Warning               CS8034  Unable to load Analyzer assembly C:\Users\username\.nuget\packages\serenity.pro.coder\6.5.4\analyzers\dotnet\cs\Serenity.Pro.Coder.dll: Could not load file or assembly 'file:///C:\Users\username\AppData\Local\Temp\VS\AnalyzerAssemblyLoader\283f9f009ad54c9598c19ac50e5db539\3\Serenity.Pro.Coder.dll' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)       xenoCore.Web                 1              Active

We have confirmed the DLL is indeed in the packages folder, and the analyzer files are there as well. We have updated Sergen, The Serenity.Pro.Coder shows up differently in the Dependency Solution explorer, it does not show anything in the dropdown list, but all the other Serenity packages do. It appears the files are on the local machine, but not being correctly loaded into either the project or Visual Studio. We could be way off, but here are some of the screenshots.

![error6](https://github.com/serenity-is/Serenity/assets/126830623/95a197bf-114f-4a95-a868-f8a73d33ef3d)
![error1](https://github.com/serenity-is/Serenity/assets/126830623/051eefcc-6f79-4940-b4e9-3922b6e673be)
![error8](https://github.com/serenity-is/Serenity/assets/126830623/e9d1e9c9-9e47-4e2b-af36-abd82430aa23)
![error3](https://github.com/serenity-is/Serenity/assets/126830623/42e3e250-9ddc-4b52-b033-7eb4e86a9965)
![error2](https://github.com/serenity-is/Serenity/assets/126830623/118b05de-94fc-4237-a2ee-a8eb42b1bb18)
![error4](https://github.com/serenity-is/Serenity/assets/126830623/f82138c3-9600-4d1a-8ccd-21090d5fbbf2)
![error5](https://github.com/serenity-is/Serenity/assets/126830623/4955880d-db29-4594-b45c-7936dfac5185)

**Comment by VictorTomaili** [comment]
Please open an issue to StartSharp repository. The `Serenity.Pro.Coder` is a premium feature.
